addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -682,4 +682,39 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
         }).then(reload);
     },
 });
+
+options.registry.WebsiteSaleProductAttribute = options.Class.extend({
+    /**
+     * @override
+     */
+     willStart: async function () {
+        const _super = this._super.bind(this);
+        this.attributeID = this.$target.closest('[data-attribute_id]').data('attribute_id');
+        return _super(...arguments);
```

Comment:
I know the code comes from the WebsiteSaleProductsItem but I wonder if you cannot write it like this :

```suggestion
        this.attributeID = this.$target.closest('[data-attribute_id]').data('attribute_id');
        return this._super(...arguments);
```